deploy/helm/kubecf/values.yaml

```diff
@@ -53,6 +53,11 @@ releases:
     stemcell:
       os: SLE_15_SP1
       version: 25.2-7.0.0_374.gb8e8e6af
```

Comment:
If we are bumping the stemcell to be consistent in all images, then it should happen [here](https://github.com/cloudfoundry-incubator/kubecf/pull/1001/files#diff-8fc2de7ef5a3d25ddfce0cd023890d0cR36). Otherwise we will still use images with different stemcells so there is no benefit in bumping only some of them here.

Reply:
True, we should change the automation for it. Why are there so many stemcell overrides when all images are anyway rebuild against the latest stemcell?